cmd/download.go

```diff
@@ -136,6 +145,9 @@ func runDownload(cfg config.Configuration, flags *pflag.FlagSet, args []string)
 	}
 
 	dir := usrCfg.GetString("workspace")
+	if solution.Team != "" {
+		dir = filepath.Join(dir, "teams", solution.Team)
```

Comment:
This is a bit confusing. In payload it is a struct here Team is a string. I understand the difference but maybe it should be TeamSlug. Or maybe have a different name in the Solution struct

Reply:
I made this change.